.github/workflows/support-frontend-post-deploy-tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Run support-frontend-post-deploy-tests
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
Keeping this workflow to run only on push to main based on the [solution ](https://github.com/orgs/community/discussions/26724)given here